docs/editor.md

```diff
@@ -33,37 +34,38 @@ const App = () => {
-It has the type `Value`, but should generally considered to be "opaque", so don't rely on its internal structure.
+This is the content that the editor will display. The data for this prop may come from any source, eg a file or database or API etc. It is a JSON representation of the entire content that should be rendered, including the external visible information, and internal information required to render the information - eg ids, versions, plugin info, etc.
+
+This information is "opaque", and is not meant to be edited directly under normal circumstances, so don't rely on its internal structure.
+
+It does not contain any presentation aspects of the data, ie no CSS is stored.
 
 ### `onChange: (newValue: Value) => void`
 
-this function is called whenever the editor has new data. You would usually store the new data back to the database in this function.
-This is not required when `readOnly` is `true`
+A callback function whenever the editor has new data. This is not required when `readOnly` is `true`
 
 ### `readOnly: boolean`
 
-if set to `true`, the content cannot be edited. You need to set this if you want to just isplay your content.
-Any code that is only used for editing won't be loaded if you using webpack or similar bundlers, so you don't have to worry about bundle size!
+If set to `true`, the content cannot be edited. Set this when using the editor to display the content. The code that is used for editing isn't loaded in this case, and hence there is a reduction in the budle size if using webpack or similar bundlers.
 
-If you set it to `false` during runtime, the editing ui will be loaded and display.
+If set to `false` during runtime, the editing UI is loaded and displays. This allows editing of the content.
 
-E.g. you can have editing capabilities directly on the public facing page, where you normaly just show the content.
-Simply check whether the current user is a publisher, display a button that will set `readOnly` to false and provide an `onChange` function to save the content
+This allows one to seamlessly switch between the display and editing modes.
+
+E.g. you can have editing capabilities directly on the public facing page, where you normally just show the content. Simply check whether the current user is a publisher, display a button that will set `readOnly` to false and provide an `onChange` function to save the content.
 
 ### `cellPlugins: CellPlugin[]`
 
-an array of `CellPlugin`s that can be used in this editor.
+An array of `CellPlugin`s that can be used in this editor.
 
-We provide both some ready-to-use plugins and an api to create custom cell plugins to display anything you want.
+'react-page' comes with some inbuilt plugins and a superb extensible system to create new plugins for displaying anything.
 
-You will usually need the [`slate`](/slate.md) plugin for rich text editing and some custom plugins.
+At the very least, the [`slate`](/slate.md) inbuilt plugin for rich text editing is needed.
```

Comment:
I would also argue that you will always want that plugin, but there are users out there of the library who actually do not use it ;-)